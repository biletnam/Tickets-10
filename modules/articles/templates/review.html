<h1><:v:pagetitle></h1>

<:v:moreaccess>

<:v:articleInfo>