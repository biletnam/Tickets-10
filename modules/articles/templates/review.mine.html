<h2>My status</h2>

<p>
It seems that you had not yet provided your own opinion about the article, though it is expected from the author.
</p>
<p>
Please choose either Approve or Decline the article. Also, add your comment - especially when declining!
</p>

<form action="" method="post">
<input type="hidden" name="f" value="<:v:_module_>/reviewarticle" />
<input type="hidden" name="article" value="<:v:id>" />
<table class="form">
  <tr>
    <th>Your opinion:</th>
    <td>
      <input type="radio" name="status" value="1" id="status1" checked />
      <label for="status1">Approve<label>
      &nbsp;
      <input type="radio" name="status" value="0" />
      <label for="status0">Decline<label>
    </td>
  </tr>
  <tr>
    <th>Your comment:</th>
    <td><textarea name="comment" rows="4" cols="60"></textarea></td>
  </tr>
  <tr>
    <td>&nbsp;</td>
    <td><input type="submit" value="Save" /></td>
  </tr>
</table>
</form>