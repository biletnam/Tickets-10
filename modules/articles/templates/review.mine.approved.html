<h2>My status</h2>

<p>
You have already Approved this article. 
No action is needed, however, you can change status of your review or withdraw it.
</p>

<form action="" method="post">
<input type="hidden" name="f" value="<:v:_module_>/reviewarticle" />
<input type="hidden" name="id" value="<:v:id>" />
<input type="hidden" name="status" value="0" />
<table class="form">
  <tr>
    <th>Your current status:</th>
    <td><strong>Approved</strong></td>
  </tr>
  <tr>
    <th>Your last comment:</th>
    <td><:v:comment></td>
  </tr>
  <tr>
    <th>Your comment for declining:</th>
    <td><textarea name="comment" rows="4" cols="60"></textarea></td>
  </tr>
  <tr>
    <td>&nbsp;</td>
    <td>
      <input type="submit" value="Decline" />
      <input type="button" value="Withdraw review" onclick="gc('Really withdraw your opinion about the article?', '?f=<:v:_module_>/deletearticlereview&id=<:v:id>');" />
    </td>
  </tr>
</table>
</form>